FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -367,7 +263,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                             </layoutDesc>
                             </objectDesc>
                             <handDesc><handNote xml:id="h1" script="Ethiopic">
-                                <ab type="script"/>
+                                <desc>Scribe <persName ref="PRS0000">አርሳንዮስ፡</persName> mentioned in <ref type="item" corresp="#coloph1"/> and <ref type="item" corresp="#coloph2"/></desc>
```

Comment:
NB new comment

Scribe is part of `handDesc`, not of `provenance`, moved here

Note that the reference to `colophon `is not a `locus `(you had `<locus target="coloph2"></locus>`), `locus `is reserved for numbers of folia, it should be a `ref type="item"`